src/core/dom.ts

```diff
@@ -122,11 +126,19 @@ export class AbstractWrapper<ElementType extends Element>
 
   findComponent<Wrapper extends ComponentWrapper<ElementType>, ElementType extends HTMLElement>(
     selector: string,
-    ComponentClass: { new (element: ElementType): Wrapper }
+    ComponentClass: WrapperClass<Wrapper, ElementType>
   ): Wrapper | null {
     const elementWrapper = this.find<ElementType>(selector);
     return elementWrapper ? new ComponentClass(elementWrapper.getElement()) : null;
   }
+
```

Comment:
Why not add some API doc description? Same for the integration testing API

Reply:
I'll clarify this point first as it's gonna have an effect on your previous comment as well. 👍